virtual_rainforest/models/animals/dummy_animal_module.py

```diff
@@ -0,0 +1,67 @@
+"""The 'dummy_animal_module.
```

Comment:
Same here, if you aren't building an explicit link which would be :mod:`~virtual_rainforest/models/animals/dummy_animal_module.py` you might as well just format module name nicely as text. I don't think there's any value in using a link here as when you build the docs they just end up being a link to the API page that they are on, so aren't of great value

Reply:
```suggestion
"""The ``dummy animal`` module.
```